geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AccumuloJoinIndex.scala

```diff
@@ -145,7 +155,7 @@ trait AccumuloJoinIndex extends GeoMesaFeatureIndex[AttributeIndexValues[Any], A
         // finally apply the arrow iterator on the resulting features
         val (iter, reduce) = ArrowIterator.configure(transformSft, this, ds.stats, None, None, hints)
         val iters = visibilityIter(indexSft) ++ Seq(filterTransformIter, rowValueIter, iter)
-        plan(iters, ArrowIterator.kvsToFeatures(), Some(reduce))
+        plan(iters, new AccumuloArrowResultsToFeatures(), Some(reduce))
```

Comment:
(Curiosity) Why the change from the object.def to having classes here?  (I'm guessing it is related to making plans serializable?)

Reply:
Yeah, so that they can be serialized